evildmp/vanilla-sphinx-test
# Fix right hand toc indentation

### Done

- Fixed RHS toc to indent correctly to an unlimited number of headings. 
- Hide first heading in TOC 

### QA
- Git checkout this branch
- run `make run` (may need to run `make install` first)
- Go to http://0.0.0.0:8001/about-repository/repertoire/
- Check the RHS toc doesn't include `Repertoire of basic content`
- Check the headings are indented correctly

### Fixes 

#13 #11
